strapi/strapi
# [breaking change] Remove the TOrigin Type Param for Attribute.UID

### What does it do?

The `TOrigin` type param for `Attribute.UID` was deprecated in V4 and is not used.

This PR aims to remove it completely (from the usage in the code & from the type gen)

Update unit tests

### Why is it needed?

Cleanup types

### How to test it?

Make sure types that depend on `Attribute.UID` stills work properly and that the type generation no longer adds the `TOrigin` type param to generated types
